test/fixtures/policy/crypto-default-encoding/policy.json

```diff
@@ -0,0 +1,14 @@
+{
+  "resources": {
+    "./parent.js": {
+      "integrity": "sha384-j4pMdq83q5Bq9+idcHuGKzi89FrYm1PhZYrEw3irbNob6g4i3vKBjfYiRNYwmoGr",
```

Comment:
this should have both \n and \r\n integrity values, otherwise machines with git set to automatically translate line endings will cause this test to fail.

Reply:
Oh, that's unfortunate. Can we use a [`.gitattributes` file](https://git-scm.com/docs/gitattributes) to avoid generating the policy programmatically? I am not sure if a local config can override it.

Reply:
probably? idk , haven't use eol= in one before